crowdnotifier-sdk/src/main/proto/qr_v3.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+package org.crowdnotifier.android.sdk.model;
+
+message QRCodePayload {
```

Comment:
Same question as above: shouldn't this be `*V3`, or at least be in a package `org.crowdnotifier.android.sdk.model.v3`

Reply:
I have changed this.